packages/app/src/cli/prompts/deploy-release.test.ts

```diff
@@ -178,7 +178,7 @@ describe('deployOrReleaseConfirmationPrompt', () => {
             },
             {
               header: 'Extensions:',
-              helperText: 'Removing extensions can permanentely delete app user data',
+              helperText: 'Removing extensions can permanently delete app user data',
```

Comment:
I happened to just notice this, it's not the reason I opened this PR but I figured it can tag along.